README.md

```diff
@@ -27,6 +27,7 @@ Please refer to the `dev-env` directory or the corresponding documents in
 
 ## Contribute
 
+See the following [document](./CONTRIBUTING.md) to contribute to the project
 See the Wiki page [Contribute to Tournesol][tournesol-wiki-contribute] for
```

Comment:
@amatissart @lfaucon 

Should we update or hide this invitation to check the Wiki?

The `CONTRIBUTING.md` will contain a lot of details too, so this raises the question why a user would click on the Wiki link instead of the Wiki? What content will be in the Wiki and not in the CONTRIBUTING.md?

I think we could be more precise by saying "See the Wiki if you want to XXX ". This will avoid having two distinct sources of truth referenced in the same document.

For instance, we could decide that all contribution instructions go to the CONTRIBUTING.md file. Or, all contribution instructions go to the CONTRIBUTING.md file expect those explaining how to compare videos, or how to be involved in social activities like brainstorming or so.

Reply:
And most of the content on the Wiki page "Contribute to Tournesol" is outdated. So I would agree to remove it for now, until the need for this wiki page is clarified.

Reply:
I have removed the link to the wiki page